engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/ExecuteJob.scala

```diff
@@ -24,65 +24,72 @@ class ExecuteJob(
       isCancellable = true,
       // TODO[MK]: make this interruptible when https://github.com/oracle/graal/issues/3590
       // is resolved
-      mayInterruptIfRunning = false
+      mayInterruptIfRunning = true
```

Comment:
Can we make it interruptible even if the issue in the comment is not resolved?

Reply:
I wonder, if we make all the execute jobs interruptible, would it break the scenario with the `executeExpression` spam, when the user quickly switches between two entries in the component browser menu?

Reply:
> Can we make it interruptible even if the issue in the comment is not resolved?

I tried to reproduce but can't anymore. I will modify the comment

Reply:
> I wonder, if we make all the execute jobs interruptible, would it break the scenario with the `executeExpression` spam, when the user quickly switches between two entries in the component browser menu?

That is actually a good point. Looks like I won't get aware from the introducing a new kind of `ExecuteJob` after all

Reply:
> > I wonder, if we make all the execute jobs interruptible, would it break the scenario with the `executeExpression` spam, when the user quickly switches between two entries in the component browser menu?
> 
> That is actually a good point. Looks like I won't get away from introducing a new kind of `ExecuteJob` after all

This has been addressed. Spamming `executeExpression` won't flood the engine with multiple `ExecuteJob`s.